zenfolio-test.customdomain,json

```diff
@@ -0,0 +1,42 @@
+{
+    "providerId": "Zenfolio",
+    "serviceId": "CustomDomain-Test",
```

Comment:
this file name has comma instead of dot in the name.

Reply:
Corrected the filename and committed/pushed the update to the branch.

Reply:
file was also renamed to match expected pattern.